middleware/updateLogin.middleware.js

```diff
@@ -0,0 +1,17 @@
+const { body } = require('express-validator');
+
+const updateValidation = [
+  body('email')
+    .isEmail()
+    .withMessage('Invalid email address')
+    .matches(/^[a-zA-Z0-9._%+-]+@(gmail\.com|yahoo\.com|hotmail\.com)$/)
+    .withMessage('Invalid email domain')
```

Comment:
This check is not necessary. Just `.isEmail` is good enough

Reply:
done on 39e6e83